Test Automation Course/src/topic7/Main.java

```diff
@@ -0,0 +1,16 @@
+package topic7;
+
+import java.io.IOException;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+
+        FileManager file = new FileManager();
+
+        file.scriereaInformatieTextInFisier("C:/tools/java.txt","Hello, my name is RobotOne!!!");
+        file.alipireaInformatieiLaFisier("C:/tools/java.txt","\nHello, my name is RobotTwo!!!");
+        file.citireDateDinFisier("C:/tools/java.txt");
```

Comment:
sugerez sa incerci sa citesti si dupa un path relativ, de ex chiar din folderul unde se afla clasa FileManager, apelezi FileReader('nume.txt') daca nume.txt este in acelasi folder il va citi, aceeasi lucreaza si cu file writer.
sa iesi din folder poti cu path ('./....')